app/controllers/checkout_controller.rb

```diff
@@ -1,7 +1,22 @@
 class  CheckoutController < ApplicationController
+  before_action :is_signed_in?
+
   def index
+      redirect_to new_user_session_path if current_user.nil?
       @address = current_user.address || Address.new(user_id: current_user.id)
       @credit_card = current_user.credit_card || CreditCard.new(user_id: current_user.id)
       @states = State.all
   end
+
+  def thank_you
+    @address = current_user.address || Address.new(user_id: current_user.id)
+    @credit_card = current_user.credit_card || CreditCard.new(user_id: current_user.id)
+    render 'checkout/thank_you'
```

Comment:
since this matches the name of the endpoint you shouldn't need to explicitly render it